packages/schematics/angular/library/files/__projectRoot__/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "<%= dasherize(packageName) %>",
   "version": "0.0.1",
   "peerDependencies": {
-    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
-    "@angular/core": "^6.0.0-rc.0 || ^6.0.0"
+    "@angular/common": "^7.0.0",
```

Comment:
Can you change this to use `angular-cli/packages/schematics/angular/utility/latest-versions.ts` instead?

Reply:
I was gonna do it, though I noticed that we are targeting that path versions in the `latest-versions.ts` should I replace the `~` with a `^` here?

Reply:
Updated